test/bin/pyutils/build_bootc_images.py

```diff
@@ -346,6 +346,100 @@ def process_image_bootc(groupdir, bootcfile, dry_run):
         os.rename(f"{bf_outdir}/bootiso/install.iso", bf_targetiso)
 
 
+def process_container_encapsulate(groupdir, containerfile, dry_run):
+    ce_path = os.path.join(groupdir, containerfile)
+    ce_outname = os.path.splitext(containerfile)[0]
+    ce_outdir = os.path.join(BOOTC_IMAGE_DIR, ce_outname)
+    ce_logfile = os.path.join(BOOTC_IMAGE_DIR, f"{ce_outname}.log")
+    ce_targetimg = os.path.join(ce_outdir, "manifest.json")
+    ce_tagname = "build_image_tag"
+    ce_tagval = f"localhost/{ce_outname}:latest"
```

Comment:
Do we need to prefix all of these variables?

Reply:
It's a copy-paste from the style of other similar functions. Strictly speaking, there's no need to prefix local vars.
Just a "safe-guard" from a copy-paste.